pkg/providers/pricing/zz_generated.pricing.go

```diff
@@ -18,9 +18,9 @@ package pricing
 
 import "time"
 
-// generated at 2023-09-11T13:06:38Z for us-east-1
+// generated at 2023-08-21T14:24:21Z for us-east-1
```

Comment:
Same comment as the generated instance types. I think we can avoid running `make docgen` or `make codegen` for this PR

Reply:
reverted